code/framework/encryption/src/main/java/io/cattle/platform/framework/encryption/Hasher.java

```diff
@@ -2,13 +2,21 @@
 
 import io.cattle.platform.util.type.Named;
 import io.github.ibuildthecloud.gdapi.model.Transformer;
+import io.cattle.platform.api.auth.LocalAuthPasswordValidator;
+import io.cattle.platform.json.JsonMapper;
+
+import javax.inject.Inject;
 
 import org.apache.commons.lang3.StringUtils;
 
 public abstract class Hasher implements Transformer, Named {
 
+    @Inject
+    JsonMapper jsonMapper;
+
     @Override
     public String transform(String value) {
+        LocalAuthPasswordValidator.validatePassword(value, jsonMapper);
```

Comment:
I wanted to call the validation code on adding an account from Admin->Accounts->Add Account.
The other functions I saw on debugging that were called on adding an account in that manner had the hashed value of password entered by the user